apps/public-docsite-resources/package.json

```diff
@@ -1,7 +1,6 @@
 {
   "name": "@fluentui/public-docsite-resources",
-  "version": "1.0.0",
-  "private": true,
+  "version": "1.0.0-beta.40",
   "description": "Fluent UI React local demo app",
```

Comment:
Likely need to have the "publishConfig": { "access": "public" } flag so the first deploy of this package would be successful

Reply:
done